flatnav_python/python_bindings.cpp

```diff
@@ -70,10 +70,10 @@ class PyIndex : public std::enable_shared_from_this<PyIndex<dist_t, label_t>> {
 
   ~PyIndex() { delete _index; }
 
-  static std::unique_ptr<PyIndex<dist_t, label_t>>
+  static std::shared_ptr<PyIndex<dist_t, label_t>>
   loadIndex(const std::string &filename) {
     auto index = Index<dist_t, label_t>::loadIndex(/* filename = */ filename);
-    return std::make_unique<PyIndex<dist_t, label_t>>(std::move(index));
+    return std::make_shared<PyIndex<dist_t, label_t>>(std::move(index));
```

Comment:
Honesty not sure why, but Pybind was complaining a lot about this being a unique pointer. I don't fully understand why it needs to be a shared pointer, but I'll look into it more.